README.md

```diff
@@ -32,6 +36,19 @@ If you select `ETHERNET` with `UBLOX_ODIN_EVK_W2` you must add this to your `tar
             }
 ```
 
+#### Mbed OS 5.9 and newer
+
+With Mbed OS 5.9 the EMAC SW was refactored and a default network 
```

Comment:
- With Mbed OS 5.9**,**  ... (so add comma)
- and a default network ... (what? something missing?)